tests/functional/prometheous/test_prometheus_validations.py

```diff
@@ -974,3 +975,78 @@ def test_restart_prometheus_glusterfs_pod(self):
             # Try to fetch metric from prometheus pod
             self._fetch_metric_from_promtheus_pod(
                 metric='heketi_device_brick_count')
+
+    @pytest.mark.tier2
+    def test_metrics_workload_on_prometheus(self):
+        """Validate metrics workload on prometheus"""
+
+        self.prefix = "autotest-{}".format(utils.get_random_str())
+        self._registry_project = self.registry_sc.get(
+            'restsecretnamespace')
```

Comment:
If my requested changes from +1015 to +1051 are applicable then this can be removed. 
```
```